pgpubsub/triggers.py

```diff
@@ -16,7 +16,11 @@ def get_func(self, model: Type[Model]):
         '''
 
     def get_declare(self, model: Type[Model]):
-        return [('payload', 'JSONB')]
+        return [
+            ('payload', 'JSONB'),
+            ('notification_context', 'JSONB'),
+            ('notification_context_text', 'TEXT'),
```

Comment:
Do we need both for the backwards compatibility reason?

Reply:
No, it's just an oversight from my side. I'll remove unused variable